code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -3,6 +3,9 @@
 	desc = "Some kind of bolt action rifle. You get the feeling you shouldn't have this."
 	icon_state = "moistnugget"
 	inhand_icon_state = "moistnugget"
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'
+	slot_flags = ITEM_SLOT_BACK
```

Comment:
why?

Reply:
Overzealous on this one, sorry about that. Reverted.